src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="info">
+    <Appenders>
+        <Console name="LogToConsole" target="SYSTEM_OUT">
+            <PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n"/>
+        </Console>
+        <File name="LogToFile" fileName="logs/app.log">
+            <PatternLayout pattern="%d %p %c:%L %m%n"/>
+        </File>
+    </Appenders>
+    <Loggers>
+        <Logger name="mate.academy" level="info">
```

Comment:
```suggestion
        <Logger name="mate.academy" level="debug">
```

Reply:
What does mean changing in this part of code? What difference between "<Configuration status="info">" and "<Logger name="mate.academy" level="debug">"?

Reply:
deference between <Configuration status="info"> and <Logger name="mate.academy" level="debug"> ?

Reply:
What difference between Configuration status="info" and Logger name="mate.academy" level="debug"?

Reply:
Please, read about logging levels to answer your question